ramls/renew-by-barcode-request.json

```diff
@@ -10,6 +10,10 @@
     "userBarcode": {
       "description": "Barcode of the user (representing the patron) the item has been loaned to",
       "type": "string"
+    },
+    "checkoutServicePointId": {
+      "description": "ID of the Service Point where the last checkout occured",
```

Comment:
"ID of the Service Point that this renewal initiates"?

Reply:
I believe it is the intention of this id to pertain to the service point of last checkin and not renewal.